README.md

```diff
@@ -110,6 +91,31 @@ The default in a deployed server is that the `ehcache.xml` configuration file is
 The manager.ehcache.config.url property in site.properties can be used to point to an `ehcache.xml` at a URL of your choice.
 
 
+### Backup and restore
+Apromore stores its data objects in two places:
+* Database: all data, except the event logs
+* Event logs which are by default located in the top-level `Event-Logs-Repository` directory
+
+As such, both need to be backed up and restored.
+* To backup h2 database, it is enough to copy across the `Manager-Repository.h2.db` file
+* To backup MySQL database, the following command may be used  (If prompted for password, enter the password of the ‘apromore’ user i.e ‘MAcri’):
```

Comment:
The password depends on the USer suplied information. You might need to change this.